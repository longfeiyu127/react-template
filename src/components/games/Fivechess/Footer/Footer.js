import React from 'react';
import './Footer.less';

export default function Footer() {
  return <footer className="g-fic-footer" />;
}
