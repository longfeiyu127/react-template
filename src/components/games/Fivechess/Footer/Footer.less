.g-fic-footer{
  position: absolute;
  bottom: 0;
  height: 0.65rem;
  width: 100%;
  background: url('../../../../assets/images/games/fivechess/footer.jpg') no-repeat;
  background-size: 100% 100%;
}