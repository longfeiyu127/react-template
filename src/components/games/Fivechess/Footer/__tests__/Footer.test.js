import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../Footer';

describe('<Footer />', () => {
  it('component render', () => {
    let footer = renderer.create(<Footer />).toJSON();
    expect(footer).toMatchSnapshot();

    footer = renderer.create(<Footer />).toJSON();
    expect(footer).toMatchSnapshot();
  });
});
