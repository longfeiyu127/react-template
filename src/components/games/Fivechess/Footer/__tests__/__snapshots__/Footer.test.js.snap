// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> component render 1`] = `
<footer
  className="g-fic-footer"
/>
`;

exports[`<Footer /> component render 2`] = `
<footer
  className="g-fic-footer"
/>
`;
